docs/chapters/architecture/SOTA-client.rst

```diff
@@ -64,3 +64,115 @@ Glossary
+this tool does not updates the host OS (bootloader, kernel, rootfs) and it
+requires applications designed to be ran in a container environment which is not
+the case in PELUX. Because of those two reasons, this solution hasn't been
+retained for PELUX.
+
+**Adaptive Autosar App**: An Adaptive Autosar application in charge of
+distributing updates across the entire car could potentially be developed in the
+future and become a standard for the industry. However, as of today, it is
+purely speculative, it would require a high stage of integration into an actual
+car and it would still require some sort of component in the PELUX side to
+apply the updates. While this is useful to keep in mind for the future, this
+can not be retained for PELUX.
+
+**Mender**: This is a block based update solution that supports rollback and
+atomic updates. It guarantees integrity and authentication security
+requirements, has a fully-featured deployment panel and a handy Yocto layer.
+Mender is easy to integrate to an embedded Linux system but at the cost of its
+lack of flexibility. Mender imposes an A/B scheme with two additional partitions
+for bootloader and data. The kernels also have to be located in the A and B
+partitions as files. The goal of PELUX being to serve as a baseline for various
+projects, we will prefer a more flexible solution such as one of those detailed
+below.
+
+**OSTree**: OSTree is an elegant file-based update mechanism that uses hard links
+to achieve in-place(no A/B partitioning) atomic updates. It is often described
+as a "git for operating systems". It currently benefits from a very large and
+active community. It has support for rollback  It integrates with Yocto easily.
+It was chosen by AGL for all of those reasons. However, OSTree suffers from some
+limitations if the rootfs to be upgraded is corrupted and since OSTree is only
+able to update file systems, it can not always upgrade kernels and can not flash
+other types of firmwares such as Bootloaders or MCUs. Hence, this solution may
+not be enough on its own depending on the needs of the project.
+
+**QtOTA**: This solution contains a set of scripts and QML APIs to easily
+integrate OSTree in a Yocto and Qt/QML system. QtOTA seems preferable over
+OSTree alone if the final system is tightly linked to a Qt architecture.
+
+**GENIVI SOTA (Aktualizr)**: GENIVI defined a modular architecture for Software
+Over The Air update deployment split into a SOTA Server, SOTA Client and
+installer. The SOTA server offers various deployment scenarios based a on VIN
+(vehicle identifiers) registry. The client side, whose current reference
+implementation is Aktualizr, can download any kind of data from the server and
+relay that to an installer. It is also worth noting that this implementation
+supports complex security mechanisms using Uptane and RVI. This architecture
+isn't enough on its own but can be integrated with something else to provide a
```

Comment:
Why isn't it enough on its own? It doesn't follow naturally from the rest of the text.